packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md

```diff
@@ -190,14 +190,14 @@ var enrollAuthenticatorOptions = new EnrollAuthenticatorOptions
 var enrollResponse = await idxAuthClient.EnrollAuthenticatorAsync(enrollAuthenticatorOptions, (IIdxContext)Session["IdxContext"]);
 ```
 
-### Step 12: Update the Email Factor Verification email template (optional)
+### 12: (Optional) Update sent email template
```

Comment:
Nitpick — I think the colon should be put after `(Optional)`, as it's not part of the step, but more part of the meta data about the step (i.e. the step number and whether it's optional both sounds like metadata)